src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -2273,6 +2832,16 @@ ucs_status_t uct_ib_mlx5_devx_mem_attach(uct_md_h uct_md,
     hdr       = UCT_IB_MLX5DV_ADDR_OF(create_alias_obj_in, in, hdr);
     alias_ctx = UCT_IB_MLX5DV_ADDR_OF(create_alias_obj_in, in, alias_ctx);
 
+    /* If received mkey is UMR key, find alias in the UMR hash map */
+    if (uct_ib_mlx5_devx_umr_mkey_is_xgvmi(packed_mkey->lkey)) {
```

Comment:
maybe this if condition is not needed, or move it inside uct_ib_mlx5_devx_umr_mkey_hash_find

Reply:
Ok, I see, I think we can remove this condition